person.rb

```diff
@@ -0,0 +1,31 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+end
```

Comment:
- [ ] Hey there! 🖐️ It's important to note that all methods declared inside a class are public by default, so there's no need to explicitly include the "public" keyword before adding public methods. 🚀📝
- [ ] As a helpful rule of thumb, when you have methods marked as private, it's considered good practice to place them at the end of the class hierarchy. 🤓🔒 This makes the class structure more organized and easier to understand, as it separates the public interface from the private implementation details. 👨‍💻👩‍💻📚

```suggestion
 def can_use_services? 
    of_age?() || @parent_permission
  end
  
  private

 def of_age?
    @age >= 18
  end
```